src/diffusers/schedulers/scheduling_edm_dpmsolver_multistep.py

```diff
@@ -126,6 +139,21 @@ def __init__(
                 f"`final_sigmas_type` {final_sigmas_type} is not supported for `algorithm_type` {algorithm_type}. Please choose `sigma_min` instead."
             )
 
+        if noise_sampling_strategy not in ["normal_distribution", "brownian_tree"]:
+            raise ValueError(
+                f"`noise_sampling_strategy` {noise_sampling_strategy} is not supported. Please choose one of `normal_distribution` and `brownian_tree`."
+            )
+
+        if noise_sampling_strategy == "brownian_tree" and not is_torchsde_available():
+            raise OptionalDependencyNotAvailable(
+                "`noise_sampling_strategy == 'brownian_tree'` but the `torchsde` library is not installed. Install it with `pip install torchsde`."
+            )
```

Comment:
For now, I've left the brownian tree sampling strategy as:
1. the quality seems a bit better with it
2. torchsde is installed when using `diffusers` if I'm not wrong!

I feel like the way I've added it doesn't bring so much trouble, but if you want, I can remove it and switch to the previous noise sampling strat!